recipes/fludashboard/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "fludashboard" %}
+{% set version = "0.3.1" %}
+{% set sha256 = "824d099574c5e13f9907eaa3f07b907e6dac2f63713b413bb8b7e310efafe1be" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win or py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
```

Comment:
Do you need `pip` here?

Reply:
I use pip in setup.py ..  When it tried to build for py2k raised error .. Now we just build for linux and osx and for py3k ..  If yoy want we can try again with no pip in requirements.